geometry_msgs/Point[] keyframe_pts
geometry_msgs/Point[] motion_pts
sensor_msgs/Image keyframe_descriptors
sensor_msgs/Image motion_descriptors
std_msgs/Float32[] kf_descriptors
std_msgs/Float32[] mv_descriptors
std_msgs/Header timestamp 
